Application/EdFi.Ods.Api/Security/AuthorizationStrategies/Relationships/Filters/RelationshipsAuthorizationFilters.cs

```diff
@@ -30,30 +30,21 @@ public static class RelationshipsAuthorizationFilters
         private static readonly Lazy<FilterApplicationDetails> _stateEducationAgencyIdToStateEducationAgencyId =
             new Lazy<FilterApplicationDetails>(() => CreateClaimValuePropertyFilter("StateEducationAgencyId"));
 
-        private static FilterApplicationDetails CreateClaimValuePropertyFilter(string propertyName)
-        {
-            return new FilterApplicationDetails(
-                $"{propertyName}To{propertyName}",
-                $"{propertyName} IN (:{propertyName})",
-                $"{{currentAlias}}.{propertyName} IN (:{propertyName})",
-                (c, w, p, jt) => w.ApplyPropertyFilters(p, propertyName),
-                (t, p) => p.HasPropertyNamed(propertyName));
-        }
-
         private static readonly Lazy<FilterApplicationDetails> _localEducationAgencyIdToStudentUSI
             = new Lazy<FilterApplicationDetails>(
                 () =>
                     new FilterApplicationDetails(
                         "LocalEducationAgencyIdToStudentUSI",
                         @"StudentUSI IN (
                             SELECT {newAlias1}.StudentUSI 
-                            FROM auth.LocalEducationAgencyIdToStudentUSI {newAlias1} 
-                            WHERE {newAlias1}.LocalEducationAgencyId IN (:LocalEducationAgencyId))",
+                            FROM auth.StudentUSIToEducationOrganizationId {newAlias1} 
+                            WHERE {newAlias1}.SourceEducationOrganizationId IN (:SourceEducationOrganizationId))",
```

Comment:
This is where you're changing the parameter name.  This parameter name is matched based on the ClaimEndpointName, defined in AggregateRootCriteriaProviderDecoratorBase.cs.  You're explicitly checking there to try to convert to the name "SourceEducationOrganizationId", but I think if you just leave this as the old parameter name of LocalEducationAgencyId then this will still function as before.  This filter is specific to claims involving LocalEducationAgencyId.

Reply:
As discussed ,  I am not doing this change due to below reason